### ISSUE TITLE
Safari: Missing visual focus and cannot tab on SAP Logo

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#5037]
<img width="1252" alt="Screen Shot 2019-10-16 at 1 42 46 PM" src="https://user-images.githubusercontent.com/45409515/66944394-18101080-f01b-11e9-9bf9-20b9c03e428d.png">

**Comment by KateChuen** [comment]
Tested on dev3. It works after enabling in safari (#5041)
